README.md

```diff
@@ -61,9 +61,38 @@ your jspm config file.
 `systemjs-hot-reloader` will automatically set `SystemJS.trace = true`, so you no longer
 need to set this manually, as with previous versions.
 
+### Worker usage
+Todays apps often use workers to move time intensive processing out of the browsers GUI thread. In such a case, use of hot-reloading is beneficial for the worker threads, too. Therefore `systemjs-hot-reloader` and [systemjs-hmr](https://github.com/alexisvincent/systemjs-hmr) can be used from workers.
+
+The following shows an example worker bootstrap script. It imports and configures `systemjs`, imports and connects the `systemjs-hot-reloader` and imports the implementation of the worker.
+```js
+importScripts('../node_modules/systemjs/dist/system.src.js', './system.config.js')
+
+System.import('systemjs-hot-reloader').then(function (connect) {
+    if (connect instanceof Function) {
+        connect()
+    }
+    System.import('WorkerImpl.js')
+})
+```
+
+Hints:
+* Using a single `systemjs` configuration (`system.config.js` in this case) for the GUI/main and the worker threads simplyfies the setup.
+* The worker is instantiated as usal: `new Worker("WorkerBootstrap.js")`
+
+### Circular dependencies / detection of entry points
```

Comment:
I actually recently updated `systemjs-hmr` so that it supports circular dependencies in the roots as well :) Check out the `findEntries` function.

The entries docs are helpful thanks, but in terms of ordering, I would prefer that the workers and entries docs came after the basic usage (including state reloading)